go.mod

```diff
@@ -5,13 +5,13 @@ go 1.22.7
 toolchain go1.22.9
 
 require (
-	github.com/avast/retry-go/v4 v4.6.0
+	github.com/avast/retry-go/v4 v4.6.1
 	github.com/google/uuid v1.6.0
 	github.com/hashicorp/go-retryablehttp v0.7.7
-	github.com/hashicorp/terraform-plugin-docs v0.20.1
-	github.com/hashicorp/terraform-plugin-framework v1.13.0
+	github.com/hashicorp/terraform-plugin-docs v0.21.0
+	github.com/hashicorp/terraform-plugin-framework v1.14.1
```

Comment:
This is what gets us write-only arguments - we have an [issue](https://linear.app/prefect/issue/PLA-1123/consider-implementing-write-only-arguments) already to look into this.